htdocs/libraries/icms/ipf/export/Handler.php

```diff
@@ -11,26 +11,79 @@
+     *
+     * @var \icms_ipf_Handler 
+     */
 	public $handler;
+        
+        /**
+         * criteria how to filter exported data
+         *
+         * @var \icms_db_criteria_Element
+         */
 	public $criteria;
-	public $fields;
-	public $format;
-	public $filename;
-	public $filepath;
-	public	$options;
+        
+        /**
+         * Fields to be exported. If FALSE then all fields will be exported
+         *
+         * @var array|bool 
+         */
+	public $fields = false;
+        
+        /**
+         * Format of the ouputed export. Currently only supports CSV
+         *
+         * @var string
+         */
+	public $format = 'csv';
+        
+        /**
+         * Name of the file to be created
+         *
+         * @var string
+         */
+	public $filename = '';
+        
+        /**
+         * Path where the file will be saved
+         *
+         * @var string 
+         */
+	public $filepath = '';
+        
+        /**
+         * Options of the format to be exported in
+         *
+         * @var array 
+         */
+	public $options = array();
+        
+        /**
+         * Output methods used for formating exported data
+         *
+         * @var bool|array
+         */
 	public $outputMethods=false;
-	public $notDisplayFields;
+        
+        /**
+         * Fields data should not be included in generated result
+         *
+         * @var array 
+         */
+	public $notDisplayFields = array();
 
 	/**
 	 * Constructor
 	 *
-	 * @param object $objectHandler IcmsPersistableHandler handling the data we want to export
-	 * @param object $criteria containing the criteria of the query fetching the objects to be exported
-	 * @param array $fields fields to be exported. If FALSE then all fields will be exported
-	 * @param string $filename name of the file to be created
-	 * @param string $filepath path where the file will be saved
-	 * @param string $format format of the ouputed export. Currently only supports CSV
-	 * @param array $options options of the format to be exported in
+	 * @param \icms_ipf_Handler         $objectHandler  IcmsPersistableHandler handling the data we want to export
+	 * @param \icms_db_criteria_Element $criteria       Containing the criteria of the query fetching the objects to be exported
+	 * @param array|false               $fields         Fields to be exported. If FALSE then all fields will be exported
+	 * @param string                    $filename       Name of the file to be created
+	 * @param string                    $filepath       Path where the file will be saved
+	 * @param string                    $format         Format of the ouputed export. Currently only supports CSV
+	 * @param array                     $options        Options of the format to be exported in
 	 */
 	public function __construct(&$objectHandler, $criteria=null, $fields=false, $filename=false, $filepath=false, $format='csv', $options=false) {
 		$this->handler = $objectHandler;
```

Comment:
Not a problem with this commit, but something to consider - the arguments' default values in the method signature do not match the declared types ($filepath is a string and the default value in the constructor is FALSE). The values assigned in the variable declarations do appear to match - public $filepath = '';